### PR TITLE
Updated NetworkDevice and constructors to have and take a new variabl…

### PR DESCRIPTION
**Pull request by T0YMAK3R**
…e, Integer index, which will be used for building an adjacency matrix for determining shortest path with Dijstra alg.

### CODE REVIEW

File: src/main/java/Objects/GameLevel.java [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 import Objects.NetworkDevices.NetworkDevice;
 
+import java.util.Iterator;
```
Comment (nnickodem) [inline src/main/java/Objects/GameLevel.java]: unused imports

File: src/main/java/Objects/NetworkDevices/NetworkDevice.java [hunk 2]
```diff
@@ -16,11 +16,12 @@
     private Integer max_packet;
     private List<String> packets;
     private String type;
+    private Integer index;
```
Comment (nnickodem) [inline src/main/java/Objects/NetworkDevices/NetworkDevice.java]: no getter method?

**Review by nnickodem — CHANGES_REQUESTED** [review]

File: src/main/java/Objects/GameLevel.java (hunk 1 above)
Comment (T0YMAK3R) [reply src/main/java/Objects/GameLevel.java]: Unused imports removed.

File: src/main/java/Objects/NetworkDevices/NetworkDevice.java (hunk 2 above)
Comment (T0YMAK3R) [reply src/main/java/Objects/NetworkDevices/NetworkDevice.java]: Created getter for the index.

**Review by nnickodem — DISMISSED** [review]

**Review by nnickodem — APPROVED** [review]

**Pull request merged**